taxonomies/ingredients.txt

```diff
@@ -10731,21 +10792,19 @@ es:lactosa y minerales de la leche
 nl:lactose en melkmineralen
 # ingredient/fr:lactose-et-minéraux-du-lait has 139 products in 5 languages @2019-04-16
 
-# Unsorted milk stuff
-
-fr:lait de montagne
-
-fr:lait entier standardisé
-
-fr:lait de bufflonne pasteurisé
 
 ################################## Yogurt related stuff ####################
 
+<en:dairy
```

Comment:
Please don't add parents to show the origin. Instead you can put a property:
from_milk:en:yes

Reply:
"dairy" occurs as allergen, that is the reason why it has been added. For me it is an isa parent. At least in dutch it works like that.